.github/command-docs.md

```diff
@@ -79,7 +79,8 @@
 | .msfs           | Provides links to MSFS support for sim issues                                                                                                                                                                                                                               | .msfsforum                                         |
 | .navdata        | Explains the use of navdata by the FlyByWire A32NX and how to check its accuracy                                                                                                                                                                                            | ---                                                |
 | .navigraphdata  | Provides help with Navigraph NavData reinstall                                                                                                                                                                                                                              | .navigraphnavdata                                  |
-| .reportedissues | Provides a link to the reported issues page within docs, either a general link, or a link for a specific search <br> Search terms can be terminated by a line break or one of the symbols . - > / \                                                                         | .issues                                            |
+| .reportedissues | Provides a link to the reported issues page within docs, either a general link, or a link for a specific search <br> Search terms can be terminated by a line break or one of the symbols . - > / \                                                                       | .issues                                           |            
+| .remoteefb            | Provides information about to the viewing the EFB remotely                                   | .refb <br> .remoteflypad <br> .rflypad           |                          
```

Comment:
Your `|` aren't lining up with the others, which you should (use spaces, not tabs, in your editor, just to be save)